src/components/CustomExtendedDataField/CustomExtendedDataField.js

```diff
@@ -94,15 +94,32 @@ const CustomFieldText = props => {
   );
 };
 
+const validateNumber = (value, max, min, numberTooSmallMessage, numberTooBigMessage) => {
+  const parsedValue = Number.parseInt(value, 10);
+  if (parsedValue > max) {
+    return numberTooBigMessage;
+  }
+  if (parsedValue < min) {
+    return numberTooSmallMessage; 
+  }
+  return undefined; 
+};
```

Comment:
Move this to util/validators.js

Reply:
So, that it can be used as 

`validate={validators.validateInterger(max, min, numberTooSmallMessage, numberTooBigMessage)}`

Note: validateNumber is too loaded name. Number in JavaScript is double precision floating point number. We specifically need to check if the input is integer.